docs/template_plugin/tests/functional/op_reference/logical.hpp

```diff
@@ -35,15 +35,21 @@ class ReferenceLogicalLayerTest : public testing::TestWithParam<RefLogicalParams
     void SetUp() override {
         const auto& params = GetParam();
         function = CreateFunction(params.opType, params.input1.shape, params.input2.shape, params.input1.type);
-        inputData = {params.input1.data, params.input2.data};
+        if (params.opType == ngraph::helpers::LogicalTypes::LOGICAL_NOT) {
+            inputData = {params.input1.data};
+        } else {
+            inputData = {params.input1.data, params.input2.data};
```

Comment:
Can we use a vector of inputs instead of two members?
It should allow to avoid such `if`s